src-tauri/src/algorithms/diffcen.rs

```diff
@@ -0,0 +1,112 @@
+use std::collections::HashMap;
+
+use crate::aux_functions::adj_matrix::convert_to_adj_matrix;
+use crate::graph::graph_ds::Graph;
+use nalgebra::DMatrix;
+
+/// Calculates the diffusion centrality of each node in the graph.
+/// Diffusion centrality is a measure of how much information a node
```

Comment:
How should the user interpret a `None` value coming out of this function?

Reply:
That will be interpreted by the global state I am working on. Basically it depends on whether the eigenvalues are real or not. None if complex result if real.

Reply:
Okay good to know. I ask the question more as a suggestion for the docs since I didn't see any context on the `Option` return type